cmd/up/controlplane/controlplane.go

```diff
@@ -117,23 +114,6 @@ local Spaces are supported. Use the "profile" management command to switch
 between different Upbound profiles or to connect to a local Space.`
 }
 
-func extractCloudFields(obj any) []string {
-	resp, ok := obj.(*controlplane.Response)
-	if !ok {
-		return []string{"unknown", "unknown", "", "", "", "", ""}
-	}
-
-	return []string{
-		resp.Name,
-		resp.Cfg,
-		resp.Updated,
-		resp.Synced,
-		resp.Ready,
-		resp.Message,
-		formatAge(resp.Age),
-	}
-}
-
 func extractSpaceFields(obj any) []string {
 	resp, ok := obj.(*controlplane.Response)
```

Comment:
We won't be operating on a `controlplane.Response` object anymore. This should use
```suggestion
	ctp, ok := obj.(spacesv1beta1.ControlPlane)
```
then extract fields from that struct.